core/src/main/scala/kafka/controller/ControllerContext.scala

```diff
@@ -431,3 +425,22 @@ class ControllerContext {
     targetState.validPreviousStates.contains(partitionStates(partition))
 
 }
+
+/**
+  * The ControllerContextSnapshot is an immutable snapshot of the ControllorContext.
```

Comment:
nit: Line 430~433 is off by one space in terms of java doc indention (the first `*` should be aligned).
Should be
```java
/**
 * ....
 */
```
Instead of
```java
/**
  * ...
  */
```

Reply:
Thanks for pointing it out. Fixed.